### PR TITLE
[player-2063]

### PR DESCRIPTION
**Pull request by aeng7**
-fixed an issue where the volume icon and aria labels were not showing the volumeOff icon when volume was set to 0

### CODE REVIEW

File: tests/components/controlBar-test.js [hunk 1]
```diff
@@ -371,6 +371,108 @@ describe('ControlBar', function () {
     expect(newVolume).toBeGreaterThan(-1);
   });
 
+  it('should display unmute volume icon when volume is set to a non-zero value', function() {
+    var mockController = {
```
Comment (jpfranco) [inline tests/components/controlBar-test.js]: Can we use `baseMockController` here so that we don't repeat this each time?

Comment (aeng7) [reply tests/components/controlBar-test.js]: Yep, I'll make this change

Comment (aeng7) [reply tests/components/controlBar-test.js]: Done

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**